### ISSUE TITLE
Shift-Extrude inward and across parralel faces, with Intersect Parallel on, flips faces

### ISSUE THREAD

**Issue by gabrielw-us** [gabrielw-us/ProBuilder-Unreal-Public-Tracker#137]
1. New Cube
1. face mode > select top face
1. ensure "Intersect Parallel is on
1. Shift-extrude downward, into cube

Result:
![image](https://cloud.githubusercontent.com/assets/5855470/22313885/e75f7b7c-e32c-11e6-8706-10b2cf8a688f.png)

**Comment by gabrielw-us** [comment]
tested in fresh project, confirmed

**Comment by arthursw** [comment]
Yes, the auto-intersection algorithm has no good way to know in which direction to re-create faces... 

It can be improved, trying to remember the normal of previously deleted faces, but maybe we can postpone that to milestone middle term? 

Intersect with parallel faces is an advanced feature anyway.

**Comment by gabrielw-us** [comment]
Ah, okay. Let's label the "Intersect Parallel" checkbox with a warning then. I don't want to remove it for AV2, but we do need people to know it isn't perfect yet, or they'll judge us badly.

I'll work on simple ideas for that when I get to the GUI stuff- still bug-testing.
